src/components/shared/OptionGroup.tsx

```diff
@@ -123,18 +123,15 @@ class OptionGroup extends React.PureComponent<
     const _children = React.Children.map(
       children,
       (option: React.ReactElement<OptionProps>, i) => {
-        let ref = this.optionsRefsSet.get(i);
-        if (!ref) {
-          ref = React.createRef<HTMLDivElement>();
-          this.optionsRefsSet.set(i, ref);
-        }
```

Comment:
revert this entire part.
I think only the keypress part change needs to be reverted